projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts

```diff
@@ -0,0 +1,519 @@
+        VAL_CAM_MC_128
+      );
+      configuration.setQuantity(CHKBOX_PROD, 2, ATTR_CAM_MC, VAL_CAM_MC_128);
+      configuration.checkPrice(
+        CHKBOX_PROD,
+        '2x($100.00) + $200.00',
+        ATTR_CAM_MC,
+        VAL_CAM_MC_128
+      );
+
+      configuration.selectProductCard(CHKBOX, ATTR_CAM_LEN, VAL_CAM_LEN_SI);
+      configuration.selectProductCard(CHKBOX, ATTR_CAM_LEN, VAL_CAM_LEN_NI);
+      configuration.checkPrice(
+        CHKBOX_PROD,
+        '$800.00',
+        ATTR_CAM_LEN,
+        VAL_CAM_LEN_SI
+      );
+      configuration.checkPrice(
+        CHKBOX_PROD,
+        '$700.00',
+        ATTR_CAM_LEN,
+        VAL_CAM_LEN_NI
+      );
+
+      configuration.clickOnNextBtn(GRP_CAM_ACC);
+      configuration.deSelectProductCard(RADGRP, ATTR_CAM_BAG, VAL_CAM_BAG_LP);
+
+      configuration.clickOnNextBtn(GRP_CAM_IAW);
+      configuration.selectAttribute(ATTR_CAM_PROF, RADGRP, VAL_CAM_PROF_Y);
+      //wait for this option to disappear
+      configuration.checkAttrValueNotDisplayed(
+        ATTR_CAM_INS,
+        DDLB_PROD,
+        VAL_CB_INS_Y2
+      );
+
+      configuration.selectProductCard(DDLB, ATTR_CAM_INS, VAL_CB_INS_P4);
+      configuration.checkPrice(
+        DDLB_PROD,
+        '$600.00',
+        ATTR_CAM_INS,
+        VAL_CB_INS_P4
+      );
+      configuration.navigateToOverviewPage();
+
+      configurationOverview.checkConfigOverviewPageDisplayed();
+      configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_MAIN, 0);
+      configurationOverview.checkGroupHeaderNotDisplayed(GRP_CAM_ACC);
+      configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_IAW, 1);
+
+      type ovContentType = {
+        name?: string;
+        value: string;
+        type: 'product' | 'simple';
+        price?: string;
+      };
+      const ovContent: ovContentType[] = [
+        {
+          name: 'Camera Body',
+          value: 'Nikon D850',
+          type: 'product',
+          price: '+ $1,500.00',
+        },
+        {
+          name: 'Memory Card',
+          value: 'SanDisk Extreme Pro',
+          type: 'product',
+          price: '2x($100.00) + $200.00',
+        },
+        {
+          name: 'Lenses',
+          value: 'Sigma 85mm F1.4 DG HS',
+          type: 'product',
+          price: '+ $800.00',
+        },
+        { value: 'Nikon AF-P DX NIKKOR', type: 'product', price: '+ $700.00' },
+        {
+          name: 'professional photographer?',
+          value: 'yes',
+          type: 'simple',
+        },
+        {
+          name: 'Insurance',
+          value: 'Pro 4 years',
+          type: 'product',
+          price: '+ $600.00',
+        },
+      ];
+      ovContent.forEach((line, idx) => {
+        configurationOverview.checkAttrDisplayed(line.name, line.value, idx);
+        configurationOverview.checkAttrPriceDisplayed(line.price, idx);
+        configurationOverview.checkAttrType(line.type, idx);
+      });
+    });
+  });
+
+  describe('Configuration Process', () => {
```

Comment:
I'm getting error in this test when asserting the number of cart items in the mini-cart.
![image](https://user-images.githubusercontent.com/4001059/115380599-06409800-a1d3-11eb-953e-54c0f4f50fc2.png)

We're reusing the e2e tests always for the same user `cpq03@sap.com`. In case of unfinished order (or parallel runs of e2e), the number of cart items might get big.

Reply:
There is a task for it: #11479